framework/db/ActiveQuery.php

```diff
@@ -728,6 +732,88 @@ public function orOnCondition($condition, $params = [])
     }
 
     /**
+     * Starts a transform relations in exists condition
+     */
+    private function buildExistsRelations()
```

Comment:
Why? Is QueryBuilder knows something about Active db components?
`buildJoinWith`, `joinWithRelations` and `joinWithRelation` builds join from relations. 
`buildExistsRelations`, `existsRelations` and `existsRelation` does the same, but builds exists where instead.